src/components/server.ts

```diff
@@ -54,6 +54,7 @@ export class Server {
         if (request.url.indexOf('pdf:') >= 0 && request.url.indexOf('viewer.html') < 0) {
             // The second backslash was encoded as %2F, and the first one is prepended by request
             const fileName = decodeURIComponent(request.url.replace('/pdf:', ''))
+            this.pdfFile = fileName
```

Comment:
You need to remove this line as well as line 77 or else the compiler complains.

Reply:
My bad!